wp-directives.php

```diff
@@ -176,31 +152,73 @@ function wpx($data)
 	$wpx = array_merge_recursive($wpx, $data);
```

Comment:
We can replace it with `array_replace_recursive` here as well.

```suggestion
	$wpx = array_replace_recursive($wpx, $data);
```